.pre-commit-config.yaml

```diff
@@ -2,8 +2,10 @@
   sha: master
   hooks:
   - id: trailing-whitespace
+    exclude: blog/migrations/
```

Comment:
I have too long line error, it even doesn't get into "max-line-length" interval.